packstream/packstream-specification-1.md

```diff
@@ -753,18 +753,18 @@ The time zone information is specified with a zone offset.
 DateTime::Structure(
     seconds::Integer,
     nanoseconds::Integer,
-    tz_offset_minutes::Integer,
```

Comment:
this is the specification with minutes.

Reply:
In the [official driver source](https://github.com/neo4j/neo4j-java-driver/blob/4.1/driver/src/main/java/org/neo4j/driver/internal/messaging/v2/ValueUnpackerV2.java#L135-L136) the last field is used as a zone offset in seconds, not minutes.

Reply:
this is a bug, it should be `tz_offset_seconds::Integer`